webview/src/experiments/components/table/Cell.tsx

```diff
@@ -107,15 +108,24 @@ export const CellWrapper: React.FC<
     cellId: string
     children?: React.ReactNode
   }
-> = ({ cell, cellId, changes }) => (
-  <div
-    {...cell.getCellProps({
-      className: cx(styles.td, cell.isPlaceholder && styles.groupPlaceholder, {
-        [styles.workspaceChange]: changes?.includes(cell.column.id)
-      })
-    })}
-    data-testid={cellId}
-  >
-    {cell.render('Cell')}
-  </div>
-)
+> = ({ cell, cellId, changes }) => {
+  const cellhasChanges = (cell.value as ValueWithChanges)?.changes
+
+  return (
+    <div
+      {...cell.getCellProps({
+        className: cx(
+          styles.td,
+          cell.isPlaceholder && styles.groupPlaceholder,
+          {
+            [styles.workspaceChange]:
```

Comment:
[I] Name needs to be updated. Personally I think it would be ok to have two different (currently duplicate) styles. One for workspace changes and another for dep changes against HEAD.